llvm/test/Transforms/PGOProfile/memprof_max_cold_threshold.test

```diff
@@ -0,0 +1,131 @@
+;; Test the -memprof-min-percent-max-cold-size flag for annotating only the
+;; largest contexts with size info.
+
+; REQUIRES: x86_64-linux
```

Comment:
I don't think we need this since we don't deal with any raw profiles in this test?

Reply:
This just got copied over from the test I cloned this from initially. Agree, I will remove it.

Reply:
There are a few of these I came across when running memprof tests for ARM for development. I'll clean up the rest.